script/plugin_tools.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ev
+
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+  export FLUTTER_CHANGED_PACKAGES=""
```

Comment:
Probably not. I am not very fluent in bash. If it is not a big deal, let's keep it?